docs/tutorials/blog.md

```diff
@@ -573,7 +573,7 @@ Quick note on that `invariant` for the params. Because `params` comes from the U
 
 We also have an invariant for the post. We'll handle the `404` case better later. Keep going!
 
-Now let's get that markdown parsed and rendered to HTML to the page. There are a lot of markdown parsers, we'll use "marked" for this tutorial because it's really easy to get working.
+Now let's get that markdown parsed and rendered to HTML to the page. There are a lot of markdown parsers, we'll use "marked" for this tutorial because it's really easy to get working. Please note that you will need to restart your server after installing.
 
 💿 Parse the markdown into HTML
 
```

Comment:
Could we leave the following message (or similar)  around line 585 ? This message is similar to the the once above when updating prisma. Leaving it after the installation also makes it easier to follow along !


>Now that marked has been installed, we will need to restart our server. So stop the dev server and start it back up again with `npm run dev`

Reply:
Ah, I totally agree! Thanks for the suggestion. Definitely makes more sense to have it placed there instead. I also like the idea of using a similar instruction to the prisma one for consistency. I'll send up those updates soon 👍🏼